src/lib/y2partitioner/ui_state.rb

```diff
@@ -168,7 +177,11 @@ def save_open_items
 
     protected
 
-    # Where to place the user within the general tree in next redraw
+    # A sort of breadcrumbs useful to know where to place the user within the general tree in the
```

Comment:
well, I know that Shakespeare enjoy reading stuff like "A sort of breadcrumbs useful" but I think we should prefer in documentation simple english :) So simple "It is useful to know" is better for me :) ( but at least I now know new english idiom :)

Reply:
I see your point. I'll try to simplify it, but the word _breadcrumb_ was key for me to understand better what `#find_tree_node` was doing.